packages/dashboard/src/customization/widgets/lineSymbol/component.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { CSSProperties } from 'react';
+import { LineWidget } from '../types';
+import { SVG_STROKE_DASHED, SVG_STROKE_DOTTED, SVG_STROKE_SOLID } from '../constants';
+import { useWidgetActions } from '~/customization/hooks/useWidgetActions';
+import { DashboardWidget } from '~/types';
+import { LineAnchor } from './lineAnchor/component';
+import { useIsSelected } from '~/customization/hooks/useIsSelected';
+
+const LineWidgetComponent: React.FC<LineWidget> = (widget) => {
+  const { update } = useWidgetActions<DashboardWidget>();
+  const isSelected = useIsSelected(widget);
+
+  const fitFull = {
+    width: '100%',
+    height: '100%',
+  };
+
+  const {
+    lineStyle = 'solid',
+    color = 'black',
```

Comment:
just curious: is it worth to just use the cloudscape design tokens ?

Reply:
+1 to cloudscape tokens, here are the token values
https://cloudscape.design/foundation/visual-foundation/design-tokens/